build.sbt

```diff
@@ -191,7 +194,16 @@ def elasticsearch : String = {
 }
 
 def commonSettings: Seq[Setting[_]] = eclipseSettings ++ Seq(
-  javacOptions ++= Seq("-encoding", "UTF-8", "-source", "1.8", "-target", "1.8", "-Xlint:unchecked", "-Xlint:deprecation", "-parameters")
+  javacOptions in Compile ++= Seq("-encoding", "UTF-8", "-source", "1.8"),
+  javacOptions in (Compile, compile) ++= Seq("-Xlint:unchecked", "-Xlint:deprecation", "-parameters")
+)
+
+// Include this into impl projects that use the message broker API
+// It overrides the production configuration to use a hardcoded Kafka broker
+// host and port rather than looking it up from the service locator.
+// See docs/running-in-conductr.md for details.
+def kafkaSettings: Seq[Setting[_]] = Seq(
+  BundleKeys.startCommand += "-Dlagom.broker.kafka.service-name=''"
```

Comment:
It happens to default to localhost:9092: https://github.com/lagom/lagom/blob/master/service/core/kafka/client/src/main/resources/reference.conf#L4